modules/systemtests/src-test/org/ejbca/ui/cli/ca/CaImportCertDirCommandSystemTest.java

```diff
@@ -191,4 +192,41 @@ public void testImportRevokedWithReasonAndTime() throws AuthorizationDeniedExcep
         assertEquals("Certificate revocation date was incorrectly imported.", new SimpleDateFormat(CaImportCertDirCommand.DATE_FORMAT).parse("2015.05.04-10:15"),
                 certificateStatus.revocationDate);
     }
+    
+    @Test
+    public void testImportFromAnotherCA() throws Exception {
+        // Import a certificate from another CA. One way to do this is to save the current CA cert, re-create the CA, then import EE cert.
+        // Get the CA cert and save it to file
+        Certificate caCert = ca.getCACertificate();
+        File caCertFile = File.createTempFile("cacert", null, tempDirectory);
+        FileOutputStream fileOutputStream = new FileOutputStream(caCertFile);
+        try {
+            fileOutputStream.write(CertTools.getPemFromCertificateChain(Arrays.asList(caCert)));
+        } finally {
+            fileOutputStream.close();
+        }
+        // Delete the CA
```

Comment:
Since the command doc says it is for old, i.e. renewed CA certificates. Wouldn't it make sense to renew the CA instead of deleting and re-creating it? See for example RenewCASystemTest for a simple example.